purview/models/distgit.py

```diff
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from neomodel import (
+    StructuredNode, UniqueIdProperty, RelationshipTo, StringProperty,
+    DateTimeProperty)
+
+
+class DistgitCommit(StructuredNode):
+    id_ = UniqueIdProperty(db_property='id')  # commit_id
+    owner = RelationshipTo('.user.User', 'OWNED_BY')  # author
```

Comment:
If this is the author of the commit, do you think "AUTHORED_BY" makes sense for this relationship?

Reply:
I do but I tried to reuse `OWNED_BY` for simplicity in our queries.

Reply:
Yeah, I guess this is the "normalized" author relationship.  :)